recipes/scikit-procrustes/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version="0.1" %}
+
+package:
+  name: scikit-procrustes
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/scikit-procrustes/scikit-procrustes-{{ version }}.tar.gz
+  sha256: 4a4c2377693a4bd57084fe644ecf140eda9ce6a472d8f42579254c80eaafc4ce
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.12
+    - scipy
```

Comment:
Are `scipy` and `matplotlib` required to build the package? If not, please remove them.